presto-cassandra/pom.xml

```diff
@@ -160,8 +160,8 @@
         </dependency>
 
         <dependency>
-            <groupId>io.prestosql.cassandra</groupId>
-            <artifactId>cassandra-server</artifactId>
```

Comment:
When this is merged, we should archive the https://github.com/prestosql/presto-cassandra-server repository.

Reply:
Also remove this:
https://github.com/prestosql/presto/blob/c1e0d245787b503142a8b8dfc9733a34d0441017/pom.xml#L1190-L1200